configure.ac

```diff
@@ -2078,6 +2078,12 @@ if test "x$unity_define" = "xyes"; then
    AC_MSG_NOTICE([Building for Unity])
 fi
 
+AC_ARG_ENABLE(unity-android-define,[  --enable-unity-android-define have UNITY_ANDROID defined], unity_android_define=$enableval, unity_android_define=no)
+if test "x$unity_android_define" = "xyes"; then
+   AC_DEFINE(UNITY_ANDROID, 1, [Building for Unity Android])
```

Comment:
I assume this define makes it's way to C# and not just C.

Reply:
Correct. We didn't have a define for Android that we could rely on since we don't build for MONODROID specifically.